src/kongComponents/model-example.jsx

```diff
@@ -33,11 +33,31 @@ export default class ModelExample extends React.Component {
 
   render() {
     let { getComponent, specSelectors, schema, example, isExecute, getConfigs } = this.props
+    let { defaultModelExpandDepth } = getConfigs()
+    const ModelWrapper = getComponent("ModelWrapper")
 
-    return <div className="example">
-      <p>Example Value</p>
+    return <div>
+      <ul className="tab">
+        <li className={ "tabitem" + ( isExecute || this.state.activeTab === "example" ? " active" : "") }>
+          <a className="tablinks" data-name="example" onClick={ this.activeTab }>Example Value</a>
+        </li>
+        { schema ? <li className={ "tabitem" + ( !isExecute && this.state.activeTab === "model" ? " active" : "") }>
+          <a className={ "tablinks" + ( isExecute ? " inactive" : "" )} data-name="model" onClick={ this.activeTab }>Model</a>
+        </li> : null }
+      </ul>
       <div>
-        {example}
+        {
+          (isExecute || this.state.activeTab === "example") && example
```

Comment:
My React is super rusty, but will this render `false` if  ` (isExecute || this.state.activeTab === "example") ` resolves to false?